lint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+docker-compose -f docker/docker-compose.yml \
```

Comment:
This script is also called in the pre-commit hook, which means the image will be built each time a commit is made. This will slow down committing and linting significantly.

Reply:
Yeah, that does make sense. Let's add a comment in the documentation that the docker images needs to be built before lint.sh works correctly